pom.xml

```diff
@@ -99,17 +102,17 @@
         <jboss-logging.version>3.2.1.Final</jboss-logging.version>
         <seata.version>2.2.0</seata.version>
         
-        <netty.version>4.1.112.Final</netty.version>
-        <bouncycastle.version>1.78.1</bouncycastle.version>
+        <netty.version>4.1.117.Final</netty.version>
+        <bouncycastle.version>1.80</bouncycastle.version>
         
-        <curator.version>5.7.0</curator.version>
+        <curator.version>5.7.1</curator.version>
         <zookeeper.version>3.9.2</zookeeper.version>
-        <audience-annotations.version>0.12.0</audience-annotations.version>
+        <audience-annotations.version>0.15.0</audience-annotations.version>
```

Comment:
- `org.apache.yetus:audience-annotations:0.12.0` is actually just a transitive dependency of `org.apache.zookeeper:zookeeper:3.9.2`.
- And since `org.apache.yetus:audience-annotations:0.14.0`, the metadata provided by this dependency is invalid on JDK8, refer to https://issues.apache.org/jira/browse/YETUS-1132 . Why do you need to change the version of `org.apache.yetus:audience-annotations`?